tools/highlight-files/highlight-files.ts

```diff
@@ -26,20 +27,38 @@ function getBazelActionArguments() {
 }
 
 if (require.main === module) {
-  // The script expects the bazel-bin path as first argument. All remaining arguments will be
-  // considered as markdown input files that need to be transformed.
-  const [bazelBinPath, ...inputFiles] = getBazelActionArguments();
-
-  // Walk through each input file and write transformed markdown output to the specified
-  // Bazel bin directory.
-  inputFiles.forEach(inputPath => {
-    const fileExtension = extname(inputPath).substring(1);
-    // Convert "my-component-example.ts" into "my-component-example-ts.html"
-    const baseOutputPath = inputPath.replace(`.${fileExtension}`, `-${fileExtension}.html`);
-    const outputPath = join(bazelBinPath, baseOutputPath);
-    const htmlOutput = highlightCodeBlock(readFileSync(inputPath, 'utf8'), fileExtension);
-
-    writeFileSync(outputPath, htmlOutput);
-  });
+    // The script expects the output directory as first argument. Second is the name of the
+    // package where this the highlight target is declared. All remaining arguments will be
+    // considered as markdown input files that need to be transformed.
+
```

Comment:
Nit: I think this new-line sneaked in. Consider removing as the comment belongs to the args retrieval.